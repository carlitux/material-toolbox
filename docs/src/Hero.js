// @flow
/* eslint-disable */
import * as React from 'react';

import Theme from 'material-toolbox/theme';
import { LayoutGrid, LayoutInner, LayoutCell } from 'material-toolbox/layout-grid';
import { Text } from 'material-toolbox/typography';

const Hero = () => (
  <Theme theme="primary" textStyle="primary" textOn="primary">
    <header>
      <LayoutGrid>
        <LayoutInner>
          <LayoutCell align="bottom" span={12}>
            <Text textStyle="display2" component="h1">
              React Material Components for the Web
            </Text>
            <Text textStyle="headline" component="span" adjustMargin>
              Create beautiful apps with modular and customizable UI components
              using React wrapper of Google's Material Components for the Web.
            </Text>
            <Text textStyle="body2" component="p">
              Note.- This library is as simple as Material Web Components is,
              Tries to follow the guide lines as much as possible. The library
              do not do any assumptions on components or tags to be used unless
              the Material Components guide does.
            </Text>
          </LayoutCell>
        </LayoutInner>
      </LayoutGrid>
    </header>
  </Theme>
);

/* eslint-enable */
export default Hero;
