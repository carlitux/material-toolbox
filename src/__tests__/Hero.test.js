import React from 'react';
import { shallow } from 'enzyme';
import Hero from '../Hero';

describe('component::Hero', () => {
  it('Should match snapshoot', () => {
    const component = shallow(<Hero />);
    expect(component).toMatchSnapshot();
  });
});
