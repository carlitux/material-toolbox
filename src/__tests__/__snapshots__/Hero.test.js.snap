// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component::Hero Should match snapshoot 1`] = `
<Theme
  textOn="primary"
  textStyle="primary"
  theme="primary"
>
  <header>
    <Grid
      fixed={false}
    >
      <Inner>
        <Cell
          align="bottom"
          span={12}
        >
          <Text
            adjustMargin={false}
            component="h1"
            textStyle="display2"
          >
            React Material Components for the Web
          </Text>
          <Text
            adjustMargin={true}
            component="span"
            textStyle="headline"
          >
            Create beautiful apps with modular and customizable UI components using React wrapper of Google's Material Components for the Web.
          </Text>
          <Text
            adjustMargin={false}
            component="p"
            textStyle="body2"
          >
            Note.- This library is as simple as Material Web Components is, Tries to follow the guide lines as much as possible. The library do not do any assumptions on components or tags to be used unless the Material Components guide does.
          </Text>
        </Cell>
      </Inner>
    </Grid>
  </header>
</Theme>
`;
